src/main/java/org/threadly/litesockets/WireProtocol.java

```diff
@@ -0,0 +1,11 @@
+package org.threadly.litesockets;
+
+/**
+ * <p>Wire protocols supported by litesockets.  This is the protocol
+ * used to communicate with. Depending on the protocol Implementors of the SocketExecuterInterface 
+ * might need to take a different action.</p>
+ * 
+ */
+public enum WireProtocol {
+  FAKE, TCP, UDP
```

Comment:
I guess if we add my suggestion for the else condition we need to handle "FAKE" as well